### ISSUE TITLE
Uncaught (in promise) TypeError: Cannot read property 'flip' of undefined

### ISSUE THREAD

**Issue by leoliu** [pqina/filepond#131]
I have every dep updated to the latest and observed this crash with filepond 3.0.2 and its plugins.

```
Uncaught (in promise) TypeError: Cannot read property 'flip' of undefined
    at imageToImageData (filepond-plugin-image-transform.js:256)
    at eval (filepond-plugin-image-transform.js:780)
```

**Comment by ipoj** [comment]
I also found this same problem last night when upload image. my package.json include

```
    "filepond-plugin-image-preview": "^3.0.0",
    "filepond-plugin-image-resize": "^2.0.1",
    "filepond-plugin-image-transform": "^3.0.0",
    "react-filepond": "^3.0.1",
```

**Comment by rikschennink** [comment]
I think installing the `filepond-plugin-image-crop` plugin should fix this temporarily (no need to set a crop aspect ratio).

I'm working on a fix in the mean time.

**Comment by rikschennink** [comment]
Just published version `3.0.1` which should fix this issue. Please let me know if it does.

**Comment by leoliu** [comment]
Thanks, that fixes the issue for me.
